packages/mobile/src/screens/collection-screen/CollectionDownloadStatusRow.tsx

```diff
@@ -68,7 +68,8 @@ export const CollectionDownloadStatusRow = (
   const isAvailableForDownload =
     !!collection &&
     (collection.has_current_user_saved ||
-      collection.playlist_owner_id === currentUserId)
+      collection.playlist_owner_id === currentUserId) &&
+    collection.access.stream
```

Comment:
should we use the `useGatedContentAccess` hook? probably technically overkill but good to stick to patterns in case access logic changes in future?